docs/UserGuide.md

```diff
@@ -478,7 +494,19 @@ Examples:
 
 ### 5.5. Current session
 
-[short overview]
+{ start of `Current session#overview` written by: Zhang Sheng Yang }
+<br/>
+<br/>
+After entering a session, which is mentioned above, you can now edit the presence and participation details
+of a student. `participate` allows you to toggle the participation status (from absent to present or from 
+present to absent), and `presence` allows you to toggle the presence status of a student(from not 
```

Comment:
Maybe the opening "(" at the end of this line could have a space between the word "student" preceding it?